examples/shardingsphere-example-generator/src/main/resources/template/jdbc/java/config/encrypt.ftl

```diff
@@ -19,7 +19,7 @@
         Properties props = new Properties();
         props.setProperty("aes-key-value", "123456");
         EncryptColumnRuleConfiguration columnConfigAes = new EncryptColumnRuleConfiguration("phone", "phone", "", "", "phone_plain", "phone_encryptor", null);
-        EncryptColumnRuleConfiguration columnConfigTest = new EncryptColumnRuleConfiguration("status", "status", "assisted_query_status", "", "string_encryptor", "string_encryptor", null);
+        EncryptColumnRuleConfiguration columnConfigTest = new EncryptColumnRuleConfiguration("status", "status", "assisted_query_status", "", "","string_encryptor", "string_encryptor", null, null);
```

Comment:
About `"","string_encryptor"`, blank could be added after `,`

Reply:
done.